api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ConceptTranslatorImpl.java

```diff
@@ -85,6 +94,11 @@ public Concept toOpenmrsType(@Nonnull CodeableConcept concept) {
 			}
 			
 			String codingSource = conceptURLToSource(coding.getSystem());
+			if (codingSource == null) {
+				if (FhirConstants.SNOMED_SYSTEM_URI.equals(coding.getSystem())) {
+					codingSource = Duration.SNOMED_CT_CONCEPT_SOURCE_HL7_CODE;
+				}
+			}
```

Comment:
Likewise, we should be able to translate a concept back from snomed-ct, even if there is no uri mapping in the fhir table.